sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -324,8 +325,9 @@ private void exportCurrentBatch() {
         } else {
           logger.log(Level.FINE, "Exporter failed");
         }
-      } catch (RuntimeException e) {
-        logger.log(Level.WARNING, "Exporter threw an Exception", e);
+      } catch (Throwable t) {
```

Comment:
Can you add a test case with a non-runtime exception? I think Mockito will allow throwing a non-runtime exception from a mock even if not declared, or otherwise there is this pattern

https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java#L235

Reply:
I have tried to add a unit test that made exporter throw a non-runtime exception. However mockito alert that `checked exception is invalid for this method!`. Thanks for your suggestion, I will try again.

Reply:
Done. I add a unit test with a non-runtime exception.